config/messages.json

```diff
@@ -0,0 +1,3 @@
+{
+  "firstJoinMessage": "Heyas! Welcome to the AwF community. Since you are new, you don't have many permissions. For those, please go to the awf.yt Discord"
```

Comment:
perhaps add a link itself, make it more obvious

Reply:
Changed to http://awf.yt